### ISSUE TITLE
How do you actaully run kicad_pos_to_cpl.py?

### ISSUE THREAD

**Issue by Bazmundi** [wokwi/kicad-jlcpcb-bom-plugin#13]
The [read me](https://github.com/wokwi/kicad-jlcpcb-bom-plugin#generating-a-jlcpcb-cpl-file) suggests you are convering a `xxx.pos` file using `kicad_pos_to_cpl.py`, but the instructions actually seem to ask one to set up to actaully generate a `xxx-pos.csv` file.  So which is it?  Given the settings, and a scan of the code, I would say the guidance needs rewording?

The settings are also set to create a top and a bottom file, I can only assume if the top of the board has the components then whatever conversion is against only the top file?  Makes sense, since I also understand JCLPCB only place components on top.  So, a little more clarity in the guidance maybe?

**Comment by lFatality** [comment]
I have used it in the following way:
1.) Create a `xxx-pos.csv` file using KiCad (`File -> Fabrication Outputs -> Component Placement (.pos)`). As the Format choose `.csv`.
2.) Use the `kicad_pos_to_cpl.py` script to convert the `xxx-pos.csv` file into a `xxx-cpl.csv` file. To do so, execute the file in a terminal like so 
```
kicad_pos_to_cpl.py xxx-pos.csv xxx-cpl.csv
```
3.) Your `xxx-cpl.csv` file should now have been generated and reside in the same location as your `xxx-pos.csv` file.

Comparing the generated `xxx-cpl.csv` file with the sample `CPL` file from JLCPCB showed a similar file structure.
I used KiCad 6.0.4. on Ubuntu 18.04.

**Comment by hattesen** [comment]
@IFatality, the description above should be merged into the [Generating a JLCPCB CPL File](https://github.com/wokwi/kicad-jlcpcb-bom-plugin#generating-a-jlcpcb-cpl-file) section of the README.md file.

Let me know if you'd like me to create a pull request for it.

**Comment by lFatality** [comment]
@hattesen Yes, it could be a nice addition. Feel free to open a pull request.
